### PR TITLE
Add Bevy Engine icon

### PR DESCRIPTION
**Pull request by Weasy666**
bevy_simpleicons_webpage_grey

**Alexa rank:** [255,544](https://www.alexa.com/siteinfo/bevyengine.org)
**GitHub stars:** [13,608](https://github.com/bevyengine/bevy)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
**Why did you pick the hex value?**
I asked in the Bevy Discord and we settled on using the background color of the website.

**Did you manually vectorize the logo?**
Yes and no. I used an existing SVG file from the [Bevy repository](https://github.com/bevyengine/bevy/tree/main/assets/branding) and modified it according to the contribution guide, incl. the optimization steps.

EDIT: Update icon with wider gaps.

### CODE REVIEW

**Comment by Weasy666** [comment]
Please don't merge, we noticed that the gaps of the icon are nearly invisible at the small scale used in the shields. We are evaluating to increase the gap and i will probably need to change the icon.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1238,6 +1238,16 @@
             "hex": "FFB80B",
             "source": "https://partnerships.betfair.com/"
         },
+        {
+            "title": "Bevy",
+            "hex": "232326",
+            "source": "https://github.com/bevyengine/bevy/blob/main/assets/branding/bevy_bird_simpleicons.svg",
```
Comment (dirien) [inline _data/simple-icons.json:1244]: Can you point here in the Bevy homepage only please. Thanks :)

**Review by dirien — CHANGES_REQUESTED** [review]
Hi @Weasy666,

thanks for the contribution. Unfortunatly there is a differenc in the eye region

image

**Comment by dirien** [comment]
> Please don't merge, we noticed that the gaps of the icon are nearly invisible at the small scale used in the shields. We are evaluating to increase the gap and i will probably need to change the icon.

Ups, saw this reply to late and started to review! 😅

**Comment by sisp** [comment]
What's the state of this PR? Is it ready for review?

File: _data/simple-icons.json [hunk 2]
```diff
@@ -1238,6 +1238,16 @@
             "hex": "FFB80B",
             "source": "https://partnerships.betfair.com/"
         },
+        {
+            "title": "Bevy",
+            "hex": "232326",
+            "source": "https://bevyengine.org/",
```
Comment (adamrusted) [inline _data/simple-icons.json:1244]:
```suggestion
            "source": "https://bevyengine.org",
```

File: icons/bevy.svg
Comment (adamrusted) [inline icons/bevy.svg:1]: Comparing the current version in this PR against the version in the bevy repo, I'm seeing the below differences. PR in black, repo in red.

image

**Review by adamrusted — CHANGES_REQUESTED** [review]
Not sure what the status is on this - but just compared PR against the Bevy repo, and found some differences.

File: _data/simple-icons.json [hunk 3]
```diff
@@ -1238,6 +1238,16 @@
             "hex": "FFB80B",
             "source": "https://partnerships.betfair.com/"
         },
+        {
+            "title": "Bevy",
+            "hex": "232326",
+            "source": "https://bevyengine.org",
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:1244]:
```suggestion
            "source": "https://github.com/bevyengine/bevy/blob/371c90f6faecf318ff66e3c6efa6f9f48781f63f/assets/branding/bevy_bird_simpleicons.svg",
```

File: icons/bevy.svg [hunk 4]
```diff
@@ -0,0 +1 @@
....079 3.064-.321 5.211-.317 1.193.001 2.369.084 3.327.332.957.249 1.829.817 1.951 1.233.122.417.053.636-.068.962-.121.326-.492.82-1.006 1.147-.514.327-1.166.613-1.845.855-1.355.485-2.797.795-3.434.855-.135.013-.189.061-.241.155a.938.938 0 0 0-.086.4c-.006.207.027.316.086.398.058.083.15.116.245.123 4.002.269 6.994-.737 9.167-1.599 2.499-1.146 4.079-3.409 4.422-5.446.214-1.277.168-1.903-.039-2.435-.207-.533-.604-1.006-1.108-1.91a5.905 5.905 0 0 1-.617-1.682l-.023-.112.089-.071c.926-.742 1.68-1.552 1.954-2.143-.562-.597-1.532-.724-2.619-.607l-.089.01-.063-.065c-.579-.59-1.297-.892-2.031-.896Zm.325 1.379a.69.69 0 0 1 .676.531.685.685 0 0 1-.497.831.688.688 0 0 1-.842-.677c0-.368.296-.673.663-.685Zm7.353 6.621h-.006a4.644 4.644 0 0 0-.231 1.253c-.033.875.056 1.408-.045 1.975-.1.567-.398 1.125-1.157 2-.901 1.041-2.33 1.848-3.981 2.12a8.586 8.586 0 0 1-.972.536c1.284.484 2.628.511 3.693.16 1.877-.616 1.732-1.058 2.573-2.346.265-.406.57-.716.877-.971.876.382 1.737.627 2.274.566.289-.669-.006-1.445-.497-2.18.402-1.252-.209-2.505-1.564-2.948a2.907 2.907 0 0 0-.964-.165Zm1.017 1.84a.506.506 0 0 1 .393.171.519.519 0 0 1-.054.728v.001a.52.52 0 0 1-.729-.041h-.001a.52.52 0 0 1 .391-.859Z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/bevy.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Bevy</title><path d="M18.8 6.271a3.149 3.149 0 0 0-.765.081c-.018.004-.033.01-.05.013-.29.288-.616.576-.963.859.115.536.308 1.053.572 1.534.51.913.93 1.402 1.175 2.036.247.633.29 1.38.06 2.737-.283 1.685-1.307 3.469-2.933 4.785 1.262-.368 2.337-1.047 3.049-1.87.769-.887 1.025-1.39 1.115-1.895.09-.506.006-1.061.04-2.003.024-.547.131-1.088.318-1.603l.04-.112.117-.015c1.052-.137 2.003-.406 2.494-.722-.158-.71-.828-1.26-1.717-1.68l-.084-.04-.021-.09c-.282-1.209-1.226-1.989-2.448-2.015Zm.61 1.494c.064 0 .126.01.185.028a.622.622 0 0 1 .396.786.622.622 0 0 1-.778.412.622.622 0 0 1-.396-.787.627.627 0 0 1 .594-.439zm-5.69-4.61c-.764-.006-1.547.3-2.22.952-.884.858-1.158 1.63-1.003 2.594l.042.265-.263-.043c-1.942-.315-4.345-.156-7.078.95.804.344 2.278.919 3.915 1.704l.52.25-.565.115C4.907 10.38 2.342 11.645 0 14.407c1.048-.082 3.189-.334 5.423-.33 1.241.002 2.465.088 3.462.346.996.258 1.903.85 2.03 1.283.128.433.055.662-.07 1-.126.34-.512.854-1.048 1.194-.534.341-1.213.638-1.92.89-1.41.505-2.91.828-3.573.89-.14.013-.197.063-.25.162a.972.972 0 0 0-.09.416c-.006.215.028.329.09.414.06.086.156.12.254.127 4.165.281 7.278-.766 9.54-1.663 2.6-1.193 4.244-3.548 4.601-5.668.224-1.329.176-1.98-.04-2.534-.216-.555-.629-1.047-1.153-1.988a6.15 6.15 0 0 1-.642-1.75l-.024-.116.092-.074c.964-.772 1.75-1.615 2.034-2.23-.585-.622-1.594-.753-2.726-.632l-.092.01-.066-.067c-.602-.614-1.35-.929-2.113-.933Zm.337 1.434a.717.717 0 0 1 .704.553.712.712 0 0 1-.517.865.713.713 0 0 1-.856-.533.713.713 0 0 1 .669-.885Zm6.677 6.669h-.006c-.14.421-.221.86-.241 1.303-.034.911.058 1.465-.047 2.056-.105.59-.414 1.17-1.204 2.08-.937 1.085-2.424 1.924-4.143 2.207a8.908 8.908 0 0 1-1.012.559c1.337.503 2.736.531 3.844.166 1.953-.64 1.802-1.101 2.677-2.442.276-.422.594-.745.913-1.01.911.398 1.808.652 2.366.59.301-.697-.006-1.505-.517-2.27.419-1.303-.218-2.606-1.628-3.068a3.043 3.043 0 0 0-1.002-.171zM22.2 13.35c.194.22.169.56-.057.758a.538.538 0 0 1-.759-.042.538.538 0 0 1 .055-.759.56.56 0 0 1 .366-.136.523.523 0 0 1 .395.179z"/></svg>
```

File: _data/simple-icons.json [hunk 5]
```diff
@@ -1238,6 +1238,16 @@
...
+            "aliases": {
+                "aka": [
+                    "Bevy Engine"
+                ]
+            }
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:1249]: Do we need the alias? The only places I can see where they refer to themselves as "Bevy Engine" are in their [website](https://bevyengine.org/)'s `<title>` and the name of [their GH org](https://github.com/bevyengine/).
```suggestion
```

**Review by PeterShaggyNoble — DISMISSED** [review]

File: _data/simple-icons.json [hunk 6]
```diff
@@ -1834,6 +1834,16 @@
...
         },
+        {
+            "title": "Bevy",
+            "hex": "232326",
+            "source": "https://github.com/bevyengine/bevy/blob/371c90f6faecf318ff66e3c6efa6f9f48781f63f/assets/branding/bevy_bird_simpleicons.svg",
+            "aliases": {
+                "aka": [
+                    "Bevy Engine"
+                ]
+            }
```
Comment (adamrusted) [inline _data/simple-icons.json:1845]:
```suggestion
            "source": "https://github.com/bevyengine/bevy/blob/371c90f6faecf318ff66e3c6efa6f9f48781f63f/assets/branding/bevy_bird_simpleicons.svg"
```

**Review by adamrusted — APPROVED** [review]

**Pull request merged**